### DISCUSSION TITLE
3.x Feedback

### DISCUSSION THREAD

**Discussion by rumboalla** [rumboalla/apkupdater#396] [General]
Post your feedback here.
Previous discussion: https://github.com/rumboalla/apkupdater/issues/383

**Comment by AnonymousWP** [comment]
Just a quick recap of my suggestions (and the last bulletpoint is a new suggestion/report I just added):

- Bring back the parent version and target version (just like with 2.x.x)
- When enabling Aptoide, it does not ignore betas. Ignoring betas only (?) works when using APKMirror and GitHub as update source
- For some reason, 2.x.x is able to detect that there's an update available for app X and Y, while on 3.x.x it's not shown
- APKUpdater reports there's an update available for Google Drive, while in fact I already updated to the latest version. It seems like APKUpdater looks at the publish date at APKMirror instead of the actual version number.

**Comment by rumboalla** [comment]
> Just a quick recap of my suggestions (and the last bulletpoint is a new suggestion/report I just added):
> 
> * Bring back the parent version and target version (just like with 2.x.x)

Added this to the features thread.
> * When enabling Aptoide, it does not ignore betas. Ignoring betas only (?) works when using APKMirror and GitHub as update source

Aptoide should be ignoring alphas and betas per settings. But of course this depends on how the apps are labeled by Aptoide.
> * For some reason, 2.x.x is able to detect that there's an update available for app X and Y, while on 3.x.x it's not shown

Are the 2.x updates correct? With what apps and sources is this happening? 
> * APKUpdater reports there's an update available for Google Drive, while in fact I already updated to the latest version. It seems like APKUpdater looks at the publish date at APKMirror instead of the actual version number.

Dates are not used anywhere for finding updates. I tried reproducing this and I couldn't. You'll have to provide more info, like exact versions, architecture, os, etc.

**Reply by Stanzilla** [reply]
Would love to get a list view exactly like in V2, only change maybe but the current and new version on two different lines so they are better visible

**Reply by rumboalla** [reply]
This is already implemented for next version, along with what's new.

**Reply by AnonymousWP** [reply]
> Are the 2.x updates correct? With what apps and sources is this happening?

To add on top of that: it's quite noticeable how since I use V3, I have _way_ more updates available via the Play Store and _way_ less updates available via APKUpdater V3. I only use APKMirror as source. It seems like the logic differs from V2. The 2.x updates are "correct" yeah, but I can't use it anymore because it's the same package name (and thus gets updated).

> Dates are not used anywhere for finding updates. I tried reproducing this and I couldn't. You'll have to provide more info, like exact versions, architecture, os, etc.

Well, the update is gone by now, but I'll let you know when it happens again. Perhaps a screen record will do? Then you have all info too.

**Comment by TPS** [comment]
I just realized UptoDown was removed as a source between v2 & v3.  Intentional?

**Reply by rumboalla** [reply]
Old implementations of Apkpure and Uptodown used scrapping so they broke all the time and performed poorly. Since they don't have open source clients, I'd have to reverse engineer their protocol.

**Reply by FreeMan-77** [reply]
Hello.

Could you add "APKCombo" to the update search source?
I have a *.apk file translated into Russian. How can I give it to you?
I just don't know how to work with GitHub.

**Reply by rumboalla** [reply]
You can open an issue with the translation. Like this one for example (https://github.com/rumboalla/apkupdater/issues/414)

**Comment by Stanzilla** [comment]
I randomly tried logoAI and it came up with this one:

<img width="643" alt="Google Chrome Beta-2023-08-21-02 28 01" src="https://github.com/rumboalla/apkupdater/assets/75278/b062f39f-4b16-4303-8696-f43b2dd08f9f">


I'm not sure how to link to it but i only use "APKUpdater" as a prompt https://www.logoai.com/logo-maker

**Reply by FreeMan-77** [reply]
I would also change the logo. Thumbs up and italics, well, such a thing.

**Reply by rumboalla** [reply]
I'll consider any icon sent, but they need to be vectors and follow the guidelines here: https://developer.android.com/develop/ui/views/launch/icon_design_adaptive

**Comment by je-vv** [comment]
I find it uncomfortable that the apps tab always calls my attention with the total number of apps (on my system dark theme the bubble is sort of magenta).  I see no need for it at all.  The bubble with the amount of available updates on the updates tab though is pretty handy.

Greetings !

**Comment by peticho** [comment]
Some sort of indicator for the download progress might be welcome. On bigger apps, I was left wondering if anything was happening at all. When I started poking around to pass the time, it cancelled the update prompt.
Thanks for the great app!
